### PR TITLE
Non-normative examples in the Issuance

### LINKED ISSUE #336: PAR Request Object claim 'state' format: special characters are allowed or not?
In the current version of the EUDI Wallet IT spec we found:

_PAR Request Object claim 'state': Unique session identifier at the client side. This value will be returned to the client in the response, at the end of the authentication. It MUST be a random string composed by alphanumeric characters and with a minimum length of 32 digits._

This definition doesn't clarify if we can use special characters or not.

This specs:
- [https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml](https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml)
- [https://www.rfc-editor.org/rfc/rfc6749.html#appendix-A.5](https://www.rfc-editor.org/rfc/rfc6749.html#appendix-A.5)
- [https://stackoverflow.com/questions/50031993/what-characters-are-allowed-in-an-oauth2-access-token](https://stackoverflow.com/questions/50031993/what-characters-are-allowed-in-an-oauth2-access-token)

define **state** as a **VSCHAR**, i.e. [^\x20-\x7E] --> from \x20 (space) to \x7E (~) .

It's possible to reference the right definition of the "alphanumeric characters" into the EUDI Wallet IT spec?

```bash
\x20: (spazio)
\x21: !
\x22: "
\x23: #
\x24: $
\x25: %
\x26: &
\x27: '
\x28: (
\x29: )
\x2A: *
\x2B: +
\x2C: ,
\x2D: -
\x2E: .
\x2F: /
\x30: 0
\x31: 1
\x32: 2
\x33: 3
\x34: 4
\x35: 5
\x36: 6
\x37: 7
\x38: 8
\x39: 9
\x3A: :
\x3B: ;
\x3C: <
\x3D: =
\x3E: >
\x3F: ?
\x40: @
\x41: A
\x42: B
\x43: C
\x44: D
\x45: E
\x46: F
\x47: G
\x48: H
\x49: I
\x4A: J
\x4B: K
\x4C: L
\x4D: M
\x4E: N
\x4F: O
\x50: P
\x51: Q
\x52: R
\x53: S
\x54: T
\x55: U
\x56: V
\x57: W
\x58: X
\x59: Y
\x5A: Z
\x5B: [
\x5C: \
\x5D: ]
\x5E: ^
\x5F: _
\x60: `
\x61: a
\x62: b
\x63: c
\x64: d
\x65: e
\x66: f
\x67: g
\x68: h
\x69: i
\x6A: j
\x6B: k
\x6C: l
\x6D: m
\x6E: n
\x6F: o
\x70: p
\x71: q
\x72: r
\x73: s
\x74: t
\x75: u
\x76: v
\x77: w
\x78: x
\x79: y
\x7A: z
\x7B: {
\x7C: |
\x7D: }
\x7E: ~
```

</body>
</html>

### PR DESCRIPTION
**Pull request by fmarino-ipzs**
## Content
This PR separates non-normative examples in json files fixing some minor typos. Moreover this PR clarify the format of the `state` parameter according to issue #336. 

- closes #336 

## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [x] English version
- [x] Example files 
- [x] Ask for review

### CODE REVIEW

**Review by grausof — APPROVED** [review]

File: docs/en/pid-eaa-issuance.rst [hunk 1]
```diff
@@ -126,62 +127,30 @@ Below a non-normative example of the PAR.
 .. code-block:: 
 
     POST /as/par HTTP/1.1
-    Host: pid-provider.example.org
+    Host: https://eaa-provider.example.org
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
    Host: eaa-provider.example.org
```

File: docs/en/pid-eaa-issuance.rst [hunk 2]
```diff
@@ -226,7 +193,7 @@ The PID/(Q)EAA Provider returns the issued ``request_uri`` to the Wallet Instanc
 .. code-block:: http
 
     GET /authorize?client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$&request_uri=urn%3Aietf%3Aparams%3Aoauth%3Arequest_uri%3Abwc4JK-ESC0w8acc191e-Y1LTC2 HTTP/1.1
-    Host: pid-provider.example.org
+    Host: https://eaa-provider.example.org
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
    Host: eaa-provider.example.org
```

File: docs/en/pid-eaa-issuance.rst [hunk 3]
```diff
@@ -248,7 +215,7 @@ The PID/(Q)EAA Provider returns the issued ``request_uri`` to the Wallet Instanc
 .. code-block:: http
 
     HTTP/1.1 302 Found
-    Location: https://start.wallet.example.org?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd&iss=https%3A%2F%2Fpid-provider.example.org
+    Location: https://start.wallet.example.org?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd&iss=https%3A%2F%2Fhttps://eaa-provider.example.org
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
    Location: https://start.wallet.example.org?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd&iss=https%3A%2F%2Feaa-provider.example.org
```

File: docs/en/pid-eaa-issuance.rst [hunk 4]
```diff
@@ -263,7 +230,7 @@ The ``client_assertion`` is signed using the private key that is created during
 .. code-block:: http
 
     POST /token HTTP/1.1
-    Host: pid-provider.example.org
+    Host: https://eaa-provider.example.org
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
    Host: eaa-provider.example.org
```

File: docs/en/pid-eaa-issuance.rst [hunk 5]
```diff
@@ -325,10 +288,14 @@ The ``client_assertion`` is signed using the private key that is created during
...
-.. code-block::
+
+A non-normative example of the Credential Request is provided below.
+
+
+.. code-block:: http
 
   POST /credential HTTP/1.1
-  Host: pid-provider.example.org
+  Host: https://eaa-provider.example.org
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
    Host: eaa-provider.example.org
```

File: examples/at-dpop-payload.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    "iss": "https://eaa-provider.example.org",
+    "sub": "d4e0bb387aa2556ff306925fdfb9a765",
+    "aud": "https://eaa-provider.example.org/credential",
+    "iat": 1715842560,
+    "exp": 1778914560,
+    "jti": "f9655ceb-c65c-4025-9378-b6672b6149bg",
+    "client_id": "47b982369791d08003a7283f059cb0d1",
+    "cnf": {
+      "jkt": "951574aee1bb7907ae1ec3109db2b225"
+    }
```
Comment (peppelinux) [inline examples/at-dpop-payload.json:11]: what about `at_hash`?

**Review by peppelinux — APPROVED** [review]

**Pull request merged**